healme.lic

```diff
@@ -133,7 +102,7 @@ class HealMe
     if @settings.empath_healing['HEAL'] && skip_parts.nil?
       spam_heal(@settings.empath_healing['HEAL'].first, @settings.empath_healing['HEAL'][1..-1])
     else
-      wounds = check_wounds
+      wounds = DRCH.check_perc_health
```

Comment:
You should be able to get away with removing `DRCH` here, since it is included above.